docs/publish.sh

```diff
@@ -101,6 +103,8 @@ cd -
 
 # Commit the new version of the docs:
 cd $STATIC_FILES_DIR
+echo "${CNAME}" > CNAME
```

Comment:
In an ideal world this wouldn't be something that happens magically in bash, but is actually a file that lives in the website folder. This is fine for now though